package.json

```diff
@@ -7,14 +7,14 @@
     "@testing-library/jest-dom": "5.16.5",
     "@testing-library/react": "13.4.0",
     "@testing-library/user-event": "13.5.0",
-    "axios": "^1.2.2",
-    "bcrypt": "^5.1.0",
+    "axios": "1.2.2",
+    "bcrypt": "5.1.0",
     "cors": "2.8.5",
-    "dotenv": "^16.0.3",
+    "dotenv": "16.0.3",
     "express": "4.18.2",
-    "immer": "^9.0.18",
```

Comment:
:D

Reply:
i don't know why is it there,...

Reply:
immer library, didn't use it

Reply:
I was referring on to you doing good job on versions. :)